util/src/main/scala/scala/scalanative/io/VirtualDirectory.scala

```diff
@@ -29,28 +28,28 @@ sealed trait VirtualDirectory {
 object VirtualDirectory {
```

Comment:
Given that VirtualDirectory appears in the `scalanative.build` it becomes part of its public API. Virtual I/O for Scala Native is actually not so virtual after all as LLVM command we invoke require everything to be physically present on disc. We should consider dropping these abstractions altogether, they are not pulling their own weight.